src/navigation/Nav.jsx

```diff
@@ -76,40 +77,49 @@ export class Nav extends React.Component {
 		} = navItems;
 		const isLoggedOut = self.status === 'prereg' || !self.name;
 		const classNames = cx('padding--all', className);
+		const proLogo = ((proDashboard.mainAccount || {}).group_photo || {}).thumb_link;
```

Comment:
I know we try not to use lodash that
But I guess this case could be way easier if we can just use lodash/get
It'll be like

get(proDashboard, 'mainAccount.group_photo.thumb_link', '')
Rather than using this || {} syntax

Reply:
We've talked through this before - `_.get` is cool, but it's designed to handle a lot of edge cases that don't apply here https://github.com/lodash/lodash/blob/4.4.2-npm-packages/lodash.get/index.js. It uses a couple dozen functions and a handful of regexes to be highly-performant and hide the complexity of those operations behind a clever interface - it's easy to use but the implementation is much harder to reason through than the `|| {}` syntax here, so I recommend avoiding it. If you would like to make this easier to read, I recommend writing a selected that takes an 'account' object and returns the `thumb_link` and name it `getLogo`, but even that might be unnecessary indirection.